### PR TITLE
Test added for FriendsPage

### PR DESCRIPTION
**Pull request by Marion34-dev**

### CODE REVIEW

**Review by joe-winter — APPROVED** [review]

File: frontend/src/pages/Friends/FriendsPage.jsx [hunk 1]
```diff
@@ -2,7 +2,6 @@
 //import { useNavigate } from "react-router-dom";
 //import { getFriends } from "../../services/friends";
 //import Friend from "../../components/Friend";
-import LogoutButton from "../../components/LogoutButton";
```
Comment (MichaelVerdon) [inline frontend/src/pages/Friends/FriendsPage.jsx:5]: Import NavBar

File: frontend/src/pages/Friends/FriendsPage.jsx [hunk 2]
```diff
@@ -33,7 +32,6 @@ export function FriendsPage() {
     return (
         <>
```
Comment (MichaelVerdon) [inline frontend/src/pages/Friends/FriendsPage.jsx:33]: Put NavBar here

**Review by MichaelVerdon — CHANGES_REQUESTED** [review]
Just import the navbar and put it at the top of the page and after I think its perfect

File: frontend/src/pages/Friends/FriendsPage.jsx (hunk 2 above)
Comment (joe-winter) [reply frontend/src/pages/Friends/FriendsPage.jsx:33]: there is not nav bar in their branch

**Pull request merged**